### ISSUE TITLE
chrome翻不了墙,开了VPN也一样,Microsoft Edge却可以,插件我全关了

### ISSUE THREAD

**Issue by PiPi231** [bannedbook/fanqiang#389]

**Comment by bannedbook** [comment]
是这里下载的chromego吗？

**Comment by PiPi231** [comment]
不是,用了好几个VPN,有chrome插件里的,还有Shadowsocks,后面这个别的浏览器都能成功翻墙,但chrome就不行,已经重装了也一样

**Comment by PiPi231** [comment]
不是,用了好几个VPN,有chrome插件里的,还有Shadowsocks,后面这个别的浏览器都能成功翻墙,但chrome就不行,已经重装了也一样

**Comment by bannedbook** [comment]
https://github.com/bannedbook/fanqiang/wiki/Chrome%E4%B8%80%E9%94%AE%E7%BF%BB%E5%A2%99%E5%8C%85
那你用这个试试看，这个专门针对chrome的

**Comment by PiPi231** [comment]
用了,还是不行,我觉得是chrome的问题

**Comment by bannedbook** [comment]
哦，有可能
重新下载个chrome重装一下，下载的chrome注意要有google的签名

**Comment by PiPi231** [comment]
谢!解决了!

**Comment by Questdream** [comment]
把所有谷歌浏览器的账号的代理相关的插件关了

**Comment by PiPi231** [comment]
谢谢老哥



---原始邮件---
发件人: "Astrosolar"<notifications@github.com&gt;
发送时间: 2019年9月26日(星期四) 下午3:29
收件人: "bannedbook/fanqiang"<fanqiang@noreply.github.com&gt;;
抄送: "Author"<author@noreply.github.com&gt;;"PiPi231"<3542772716@qq.com&gt;;
主题: Re: [bannedbook/fanqiang] chrome翻不了墙,开了VPN也一样,Microsoft Edge却可以,插件我全关了 (#389)



把所有谷歌浏览器的账号的代理相关的插件关了

**Comment by czhmarco** [comment]
我把插件都删了，还是不行

**Comment by Essentiall** [comment]
同样问题，Chrome不能翻但无痕可以翻，ie和edge都能翻。重装不行，关闭插件代理也没用

**Comment by CongfengCai** [comment]
请问楼主是怎么解决的

**Comment by Serendipity-LJ** [comment]
> 谢!解决了!

请问是如何解决的，我下了clash,开了科学之后，edge浏览器可以翻出去，但是我原先自己有的chrome浏览器还是不能科学上网。

**Comment by Mask0ff** [comment]
rutracker的插件 删了就行了 一般都是浏览器插件的问题
